.github/workflows/release.yml

```diff
@@ -6,6 +6,10 @@ on:
       packageVersion:
         description: "The version to publish (patch, minor, current)"
         required: true
+      dryRun:
+        description: "Do a dry run to preview instead of a real release (true/false)"
+        required: true
```

Comment:
nit: this might be better as a "choice" for an easy toggle. https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/